handsontable.d.ts

```diff
@@ -1181,18 +1182,13 @@ declare namespace Handsontable {
     type UndoRedoAction = UndoRedoAction.Change | UndoRedoAction.InsertRow | UndoRedoAction.RemoveRow | UndoRedoAction.InsertCol | UndoRedoAction.RemoveCol | UndoRedoAction.Filter;
 
     interface Formulas extends Base {
-      dataProvider: DataProvider;
-      eventManager: EventManager;
-      sheet: Sheet;
-      undoRedoSnapshot: UndoRedoSnapshot;
+      staticRegister: object;
```

Comment:
should `staticRegister` be in TS definition file if it's `@private`?

Reply:
Nope ✔️